source/transactions/transactions.rst

```diff
@@ -1383,6 +1388,7 @@ durable, which achieves the primary objective of avoiding duplicate commits.
 **Changelog**
 -------------
 
+:2019-10-21: Specify that a commit error can have two error labels
```

Comment:
I think this also warrants bumping the spec's patch version from 1.5.3 to 1.5.4.